types/office-js-preview/index.d.ts

```diff
@@ -17746,11 +17746,17 @@ declare namespace Office {
          *
          * **Errors**:
          *
-         * - `HTTPRequestFailure`: The request has failed. Please look at the diagnostics object for the HTTP error code.
+         * - `GenericTokenError: An internal error has occurred.` - In Exchange Online environments, this error occurs when the token can't be retrieved because legacy Exchange tokens
+         * for Outlook add-ins are turned off. We recommend using NAA as a single sign-on solution for your add-in. For guidance on how to implement NAA, see
+         * {@link https://learn.microsoft.com/office/dev/add-ins/develop/enable-nested-app-authentication-in-your-add-in | Enable SSO in an Office Add-in using nested app authentication}.
          *
-         * - `InternalServerError`: The Exchange server returned an error. Please look at the diagnostics object for more information.
+         * - `HTTPRequestFailure: The request has failed. Please look at the diagnostics object for the HTTP error code`.
          *
-         * - `NetworkError`: The user is no longer connected to the network. Please check your network connection and try again.
+         * - `InternalServerError: The Exchange server returned an error. Please look at the diagnostics object for more information`. - In Exchange Online environments,
+         * this error occurs when the token can't be retrieved because legacy Exchange tokens for Outlook add-ins are turned off. We recommend using NAA as a single sign-on solution for your add-in.
+         * For guidance on how to implement NAA, see
+         * {@link https://learn.microsoft.com/office/dev/add-ins/develop/enable-nested-app-authentication-in-your-add-in | Enable SSO in an Office Add-in using nested app authentication}.
+         * - `NetworkError: The user is no longer connected to the network. Please check your network connection and try again.`
```

Comment:
Need an extra line here?

```suggestion
         *
         * - `NetworkError: The user is no longer connected to the network. Please check your network connection and try again.`
```